consensus/enclave/impl/src/lib.rs

```diff
@@ -697,20 +708,79 @@ impl ConsensusEnclave for SgxConsensusEnclave {
             &fee_public_key.view_public_key,
         );
 
-        let fee_outputs = total_fees_by_token_id
+        // Compute the total fees for each known token id.
```

Comment:
What' the domain being measured in "Total" here? 

Is it the cumulative fees in the transactions being considered?

Reply:
yeah the cumulative fees in the transactions of this block that we are forming